### ISSUE TITLE
AnalyticalTable - ACC: Table header read twice

### ISSUE THREAD

**Issue by priti1234567** [UI5/webcomponents-react#5819]
### Describe the bug

On accessing the table rows header items are read twice.

Affected Component
AnalyticalTable

### Isolated Example

https://stackblitz.com/edit/github-wv3ad2-gzokbg?file=src%2FApp.tsx

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

The table header should be read once.

Screenshot :

![image](https://github.com/SAP/ui5-webcomponents-react/assets/29221992/edb7b5c4-56ee-4405-bab1-5e4f76d0f6f9)

### UI5 Web Components for React Version

18.3.1

### UI5 Web Components Version

1.120.4

### Browser

Chrome

### Operating System

Microsoft Windows 11 x64

### Additional Context

Screen Reader: JAWS 2023.2307.37 ILM 64bit for Windows

### Priority

Medium

### Organization

CBC

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by surjadeep** [comment]
Any update on this issue ?

**Comment by Lukas742** [comment]
Hi all,

duplicate announcements were made because we added the header label to the `aria-label` of the cell, because Apple VoiceOver otherwise won't announce the column name. With the linked PR, we're now using `aria-labelledby` relations only for Apple devices, so NVDA and JAWS can rely soley on the accessibility tree to derive the column name.
